save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FiltersRow.kt

```diff
@@ -53,7 +55,12 @@ external interface FiltersRowProps : Props {
 @Suppress("LongMethod", "TOO_LONG_FUNCTION")
 private fun testExecutionFiltersRow(
 ) = FC<FiltersRowProps> { props ->
-    val(filters, setFilters) = useState(props.filters)
+    val (filters, setFilters) = useState(props.filters)
```

Comment:
Same question here - why can't we use `props.filters` everywhere in this component?